hr_expense_advance_clearing/views/hr_expense_views.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think this is needed and can be removed

Reply:
Done